### PR TITLE
Document workflow for using Remotion Lambda with AWS SQS

### PR DESCRIPTION
**Pull request by alexfernandez803**
Fixes issue from https://github.com/remotion-dev/remotion/issues/1619

---

<details>
<summary>
<b>IssueHunt Summary</b>
</summary>

### Referenced issues

This pull request has been submitted to:
- [#1619: Create an article explaining how to use Remotion Lambda with SQS](https://issuehunt.io/repos/274495425/issues/1619)
---
</details>

### CODE REVIEW

File: packages/docs/docs/lambda/aws-sqs-lambda-integration.md [hunk 1]
```diff
@@ -0,0 +1,368 @@
...
+      batchSize: 1,
+      maxBatchingWindow: Duration.minutes(5),
+      reportBatchItemFailures: true, // default to false
+    })
+  );
+  ```
+
+### Interacting with the API
+
+The API requires an authorization token to interact with it. To obtain the token, first go to the Serverless dashboard to retrieve outputs such as `apigw-sqs-app-stack.region`, `apigw-sqs-app-stack.userPoolClientId`, and `apigw-sqs-app-stack.userPoolId`, which are used to authenticate with Cognito. If you do not have a frontend application, you can create a user and an authentication token manually for the API by following this [guide](docs/lambda/without-iam/example#test-your-endpoint).
```
Comment (JonnyBurger) [inline packages/docs/docs/lambda/aws-sqs-lambda-integration.md]: Does this require the Serverless framework as well? Can we make it not require that?

**Comment by JonnyBurger** [comment]
Thanks a lot @alexfernandez803!
I made some changes, can you go through the new document and verify that it is all correct?
https://remotion-git-fork-alexfernandez803-1619-sqs-remotion-remotion.vercel.app/docs/lambda/sqs

If so, then I'll merge it.

**Comment by alexfernandez803** [comment]
Looks good, I have another update regarding the reference of AWS documentation.

**Comment by alexfernandez803** [comment]
I've just pushed it.

File: packages/docs/docs/lambda/aws-sqs-lambda-integration.md (hunk 1 above)
Comment (alexfernandez803) [reply packages/docs/docs/lambda/aws-sqs-lambda-integration.md]: I've pushed an update on this.

**Comment by alexfernandez803** [comment]
I've added some updates which should conclude this.

**Comment by JonnyBurger** [comment]
Nice! 🙌 Thanks a lot!

@MehmetAdemi You can reward this!

**Pull request merged**